src/software/thunderscope/robot_diagnostics/robot_info.py

```diff
@@ -290,28 +301,53 @@ def disconnect_robot(self):
             time_since_last_robot_status
             > DISCONNECT_DURATION_MS * SECONDS_PER_MILLISECOND
         ):
-            self.reset_ui()
+            self.__reset_ui()
 
-    def reset_ui(self):
+    def __reset_ui(self):
         """
         Resets the UI to the default, uninitialized values
         """
         self.robot_model.setPixmap(self.bw_vision_pattern)
 
         self.breakbeam_label.update_breakbeam_status(None)
 
-    def update_ui(self, motor_status, power_status, error_codes):
+    def __update_stop_primitive(self, is_running: bool):
+        """
+        Updates the stop primitive label based on the current running state
+        :param is_running: if the robot is running currently
+        """
+        self.stop_primitive_label.setText("RUN" if is_running else "STOP")
+        self.stop_primitive_label.setStyleSheet(
+            f"background-color: {'green' if is_running else 'red'}; border: 1px solid black;"
+        )
+
+    def __update_ui(self, robot_status):
         """
         Receives important sections of RobotStatus proto for this robot and updates widget with alerts
         Checks for
             - Whether breakbeam is tripped
             - If there are any motor faults
-            - If Battery Voltage is too low
+            - Battery voltage, and warns if it's too low
             - If this robot has errors
-        :param motor_status: The motor status message for this robot
-        :param power_status: The power status message for this robot
-        :param error_codes: The error codes of this robot
+            - If the robot is stopped or running
+            - The primitive and world loss rates
+        :param robot_status: The robot status message for this robot
         """
+        logging.debug(robot_status)
```

Comment:
Seems to be left over from debugging

Reply:
oh yeah, I didn't make this change so must've been from the robocup branch